lib/tasks/index.js

```diff
@@ -0,0 +1,9 @@
+var post = require('./post/post');
```

Comment:
`require('./post/post')...` are all invalid paths now as you have taken out sub folders